includes/Admin/Settings.php

```diff
@@ -62,18 +62,20 @@ public function __construct( bool $is_connected ) {
 	 * @since 3.0.7
 	 */
 	private function build_menu_item_array( bool $is_connected ): array {
+
+		if ( ! $is_connected ) {
+			return [ Settings_Screens\Advertise::ID => new Settings_Screens\Advertise() ];
+		}
+
 		$advertise  = [ Settings_Screens\Advertise::ID => new Settings_Screens\Advertise() ];
 		$connection = [ Settings_Screens\Connection::ID => new Settings_Screens\Connection() ];
 
-		$first = ( $is_connected ) ? $advertise : $connection;
```

Comment:
This was intentionally done. 
In case the website is not connected, the connection tab should be shown and otherwise, advertise tab should be shown. Please revert this change